argo-cd/helm-create.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+helm init
+kubectl create namespace argocd
+kubectl create serviceaccount tiller --namespace kube-system
+kubectl create -f tiller-clusterrolebinding.yaml
+helm init --service-account tiller --upgrade
+./create.sh $1
```

Comment:
I didn't put these together because helm init frequently times out or takes time to start.  This script exists in the develop_gke folder as install_tiller.sh and the rbac file is in the develop_gke/tiller folder and takes that in.

Reply:
Also if you're installing argocd with helm, you shouldn't need to make a namespace here